packages/node/test/integration/install.spec.ts

```diff
@@ -62,12 +63,27 @@ describe("Node method follows spec - install", () => {
           makeInstallCall(nodeB, msg.data.appInstanceId);
         });
 
+        // FIXME: still no symmetric events -- nodeB will never emit an
```

Comment:
This event should emit

Reply:
It does not at the moment, this test fails if this is commented out.